src/api/statements/LodexReduceQuery.js

```diff
@@ -27,8 +34,22 @@ export const createFunction = MongoClientImpl =>
         const collName = String('mp_').concat(
             hashCoerce.hash({ reducer, fields }),
         );
+
+        const handleDb = await MongoClientImpl.connect(
```

Comment:
The connection is opened but never closed. This is a memory leak.

You'd better inject a connection managed outside.

Reply:
ezs does not allow to do that.

Reply:
I think it's possible with ezs, using the configuration object, and `getParam`.